lib/components/topbar/topbar.less

```diff
@@ -398,6 +399,20 @@
         padding: var(--_tb-searchbar-p);
     }
 
+    & &--skip-link {
+        &:not(:focus) {
+            &:extend(.v-visible-sr);
+        }
+
+        background-color: var(--theme-secondary-100);
+        border-bottom: var(--_tb-bt);
```

Comment:
The skip link now includes a bottom border that replicates the top border on the topbar. When the skip link is focused, the topbar will remove it's top border in favor of this border. (see the next comment on line ~442)